### PR TITLE
chore: initial release

### PR DESCRIPTION
**Pull request by shubham-stepsecurity**
@varunsh-coder please review the changes!

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (varunsh-coder) [inline .github/CODE_OF_CONDUCT.md:1]: we can remove this file

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Configuration for request-info - https://github.com/behaviorbot/request-info
```
Comment (varunsh-coder) [inline .github/config.yml:1]: we can remove this file

File: .github/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Config for https://github.com/apps/release-drafter
```
Comment (varunsh-coder) [inline .github/release-drafter.yml:1]: do we need this with our release action?

File: .github/workflow-details.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (varunsh-coder) [inline .github/workflow-details.json:1]: do we need this?

File: .github/workflows/add-release-tag.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
+on:
```
Comment (varunsh-coder) [inline .github/workflows/add-release-tag.yml:1]: dont think we need this

File: .github/workflows/add-test-tag.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
+on:
```
Comment (varunsh-coder) [inline .github/workflows/add-test-tag.yml:1]: dont think we need this

File: .github/workflows/check-warnings.yml [hunk 7]
```diff
@@ -0,0 +1,46 @@
+on:
```
Comment (varunsh-coder) [inline .github/workflows/check-warnings.yml:1]: this uses a slack web hook. not sure if we need this

File: .github/workflows/sync-workflows.yml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+on:
```
Comment (varunsh-coder) [inline .github/workflows/sync-workflows.yml:1]: there are many workflows that we dont need

File: README.ja.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+        if: env.WORKFLOW_CONCLUSION == 'failure' # 失敗を通知する場合
+```
+
+### Success
+![Success](https://raw.githubusercontent.com/technote-space/workflow-conclusion-action/images/success.png)
+
+すべてのジョブが正常だったため、Slackアクションはスキップされます。
+
+### Failure
+![Failure](https://raw.githubusercontent.com/technote-space/workflow-conclusion-action/images/failure.png)
```
Comment (varunsh-coder) [inline README.ja.md:83]: we should update the references to technote-space/

File: README.md [hunk 10]
```diff
@@ -1 +1,85 @@
...
+          # status: ${{ env.WORKFLOW_CONCLUSION }} # neutral, success, skipped, cancelled, timed_out, action_required, failure
+          status: failure
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        if: env.WORKFLOW_CONCLUSION == 'failure' # notify only if failure
+```
+
+### Success
+![Success](https://raw.githubusercontent.com/technote-space/workflow-conclusion-action/images/success.png)
```
Comment (varunsh-coder) [inline README.md:78]: same as above

File: _config.yml [hunk 11]
```diff
@@ -0,0 +1,4 @@
+theme: jekyll-theme-architect
+
+# Google Analytics
+google_analytics: UA-78163306-3
```
Comment (varunsh-coder) [inline _config.yml:4]: not sure what this is for

File: package.json [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+    "name": "Technote",
+    "email": "technote.space@gmail.com",
+    "url": "https://technote.space"
+  },
+  "files": [
+    "dist",
+    "action.yml"
+  ],
+  "scripts": {
+    "build": "rm -rdf dist && rollup -c",
```
Comment (varunsh-coder) [inline package.json:28]: do we need to add a script to generate dist?

Comment (shubham-stepsecurity) [reply package.json:23]: We can build through ncc as well, but due to pre-commits defined by husky It allows code only in camel-case format. Hence making it difficult for user to commit.

File: .github/workflows/update-dependencies.yml [hunk 13]
```diff
@@ -0,0 +1,80 @@
+on:
```
Comment (varunsh-coder) [inline .github/workflows/update-dependencies.yml:1]: we can remove this

File: .husky/commit-msg [hunk 14]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (varunsh-coder) [inline .husky/commit-msg:1]: we can remove husky

**Review by varunsh-coder — APPROVED** [review]

**Pull request merged**